hooks/boot/30-upgrade.rb

```diff
@@ -4,3 +4,9 @@
   "Run the steps necessary for an upgrade such as migrations, rake tasks, etc.",
   :default => false
 )
+app_option(
+  '--disable-resolve-mismatches',
```

Comment:
@stbenjam, yeah, i based it off of the disable-system-checks, but am totally open to opinions for consistency and clarity. :)

Thanks for reviewing!!